arangod/RocksDBEngine/RocksDBMetadata.cpp

```diff
@@ -343,6 +342,10 @@ Result RocksDBMetadata::serializeMeta(rocksdb::WriteBatch& batch,
         }
       }
     } else {
+      output.clear();
+      rocksdb::SequenceNumber seq = rcoll->serializeRevisionTree(output, maxCommitSeq);
```

Comment:
why isn't this  using `lastSerializedRevisionTree` ?

Reply:
will this Delete be executed every time we sync ?

Reply:
@graetzer It will not be executed every time we sync. When we do this, we change the stored sequence for last serialized to max so it will never trip the condition as "needs to serialize" again. I think it makes sense to keep that logic inside `serializeRevisionTree` rather than move it to `lastSerializedRevisionTree`. Would you agree?